packages/venia-concept/src/components/FilterModal/FilterList/filterDefault.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Icon from 'src/components/Icon';
+import Checkmark from 'react-feather/dist/icons/check';
+import classify from 'src/classify';
+import defaultClasses from './filterDefault.css';
+
+class FilterDefault extends Component {
+    static propTypes = {
+        classes: PropTypes.shape({
+            root: PropTypes.string,
+            icon: PropTypes.string,
+            iconActive: PropTypes.string
+        }),
+        isActive: PropTypes.bool,
+        toggleOption: PropTypes.func,
+        value_string: PropTypes.string,
+        label: PropTypes.string,
+        group: PropTypes.string
+    };
+
+    render() {
+        const {
+            value_string,
+            toggleOption,
+            label,
+            classes,
+            group,
+            isActive
+        } = this.props;
+
+        const iconClassName = isActive ? classes.iconActive : classes.icon;
+
+        return (
+            <button
+                className={classes.root}
+                value={value_string}
+                data-group={group}
+                title={label}
+                onClick={toggleOption}
+            >
+                <span className={iconClassName}>
+                    {isActive && <Icon src={Checkmark} size={14} />}
+                </span>
+                <span
+                    dangerouslySetInnerHTML={{
+                        __html: label
+                    }}
```

Comment:
Why are we dangerously setting inner HTML here?

Reply:
Some filters (like price) return HTML as their label value.

![Screen Shot 2019-03-29 at 8 31 00 AM](https://user-images.githubusercontent.com/11479290/55216451-09881600-51fd-11e9-9d1a-3a62503c6408.png)

Feel free to let me know if there is a better approach that we should use (like stripping HTML)

Reply:
That's... an odd choice. @jimbo is that just an issue with our sample data or is there some practice of defining classes/html as copy at Magento?

Reply:
Fixed in latest commit:
- Added stripHtml function
- Removed Dangerously setting HTML functions

GraphQL only returns the HTML version of the label. There is no raw string label option implemented. So this needs to be fixed with sample data or GraphQL team.

Reply:
@codeAdrian Great, thank you. I'll make sure this gets brought up to the GraphQL team.